lib/include/openamp/rpmsg.h

```diff
@@ -129,6 +129,19 @@ struct rpmsg_device_ops {
 	int (*get_tx_buffer_size)(struct rpmsg_device *rdev);
 };
 
+/**
+ * struct rpmsg_device_trace - trace operation for debugging
+ *
+ * This structure is used by the rpmsg device to trace the various events.
+ *
+ * @get_tx_buffer: trace callback, called when get a tx buffer
+ * @release_tx_buffer: trace callback, called when release a tx buffer
+ */
+struct rpmsg_device_trace {
+	void (*get_tx_buffer)(struct rpmsg_device *rvdev, void *hdr);
```

Comment:
What these callbacks are expected to trace ?

Reply:
rpmsg_get_tx_payload_buffer and rpmsg_release_tx_buffer